packages/cloud-cognitive/src/components/BreadcrumbWithOverflow/BreadcrumbWithOverflow.js

```diff
@@ -372,7 +372,9 @@ BreadcrumbWithOverflow.propTypes = {
   /**
    * overflowAriaLabel label for open close button overflow used for action bar items that do nto fit.
    */
-  overflowAriaLabel: PropTypes.string.isRequired,
+  overflowAriaLabel: PropTypes.string.isRequired.if(
+    ({ breadcrumbs }) => breadcrumbs.length > 1
```

Comment:
I don't think I get this. The BreadcrumbOverflow component hasn't changed, so how come the logic for when an overflow aria-label is required has changed? Was it incorrect before? The comment 'title always shown' doesn't really seem to apply here, because that's a detail of the PageHeader implementation and not part of the BreadcrumbWithOverflow component logic.

Reply:
I think the logic was incorrect. 

When there is only one item supplied, the overflow is never shown.